lib/button/version.rb

```diff
@@ -1,3 +1,3 @@
 module Button
-  VERSION = '2.4.0'.freeze
+  VERSION = '2.5.0'.freeze
```

Comment:
Mind backing this out to `2.4.0`?  I can cut a release in another change if you'd like.

Reply:
Sure 👍